### DISCUSSION TITLE
Releases & Tags

### DISCUSSION THREAD

**Discussion by ElNovi** [abap2xlsx/abap2xlsx#1157] [Ideas]
Hi,

i came across company that uses old version of SAP2XLSX from the SAPLink days. I want to update it, but now i don't if i should stick with tags (latest is 7.16.0 from year ago) or that i should pull from the main repo ...

Thing is i want the most stable version and i don't know if the main repo is the right thing to pull from ... because this upgrade can be impactful on the current development that runs on production system with the old ABAP2XLSX version and i aiming to minimize the risk. 

So should i stick with tags? Is there plan to continue with the tags or releases or newer version will be on main repo only?

**Comment by larshp** [comment]
From my perspective, its up to the users to decide what to consider stable. Changes are typically not merged if they are not considered okay/stable.

Even if there is no tags at all, its always possible to do a fork and update that according to some process or never update it.

Its also possible to install multiple versions of abap2xlsx, eg. each team can have a version that they are responsible for updating, or slowly migrate each piece of logic one by one to a newer version. I dont mind duplicating code if there is a process for updating it, and clear ownership/responsibilities.

When https://github.com/abap2xlsx/abap2xlsx/pull/872 is merged, together with more work, it will be possible to trigger the logic on pull request, ie. an upgrade of abap2xslx can be tested before reaching a central development system in customer landscape.
